phantomas/mr_simul/tests/test_synthetic.py

```diff
@@ -0,0 +1,45 @@
+from phantomas.mr_simul.synthetic import *
```

Comment:
Could you import just the definitions needed in the script (rather than `import *`)?